src/components/pages/Reimbursements/Reimbursement.scss

```
@@ -0,0 +1,68 @@
+
+.reimbursements-page {
+  .header {
+    display: flex;
+    justify-content: space-between;
+    margin-top: rem(34px);
+    padding-bottom: rem(36px);
+
+    &-title {
+      @include title3();
+    }
+
+    &-initialize-filters-button {
+      @include unstyle-button;
+      @include bold;
```

Hum, je serais curieux de savoir pourquoi `bold` seul a été créé. L'idée de ces mixins fonts était d'éviter d'avoir pas mal de fonts incohérentes (du style un coup en 16px  et gras, un coup en 16px et moyen gras, dans des contextes similaires). Si on rencontrait un nouveau style de police, on papotait avec les UI pour savoir quel style déjà existant on appliquait, pour homogénéiser.

Tout ça pour dire que dans ce cas de figure j'aurais privilégié `body-important` à la place de `bold` puisque c'est ce qui est fait sur les autres boutons de réinitialisation de filtres.
```suggestion
      @include body-important;
```
À voir si on ne pourrait pas supprimer cette mixin bold pour éviter d'en induire d'autres en erreur.

---

en fait je crois qu'il y a un mixin button pour ça en fait

---

=> utilisation de la classe tertiary-button